wac.html

```diff
@@ -280,9 +289,9 @@ <h2 property="schema:name">Status of This Document</h2>
             <div property="schema:description" datatype="rdf:HTML">
               <p>This section describes the status of this document at the time of its publication.</p>
 
-              <p>This document was published by the <a href="https://www.w3.org/community/solid/">Solid Community Group</a> as a Draft. The sections that have been incorporated have been reviewed following the <a href="https://github.com/solid/process">Solid process</a>. However, the information in this document is still subject to change. You are invited to <a href="https://github.com/solid/web-access-control-spec/issues">contribute</a> any feedback, comments, or questions you might have.</p>
+              <p>This document was published by the <a href="https://www.w3.org/community/solid/">Solid Community Group</a> as an <em>Version 1.0.0-cr.1</em>. The sections that have been incorporated have been reviewed following the <a href="https://github.com/solid/process">Solid process</a>. However, the information in this document is still subject to change. You are invited to <a href="https://github.com/solid/web-access-control-spec/issues">contribute</a> any feedback, comments, or questions you might have.</p>
 
-              <p>Publication as a Draft does not imply endorsement by the <abbr title="World Wide Web Consortium">W3C</abbr> Membership. This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.</p>
+              <p>Publication as an <em>Version 1.0.0-cr.1</em> does not imply endorsement by the <abbr title="World Wide Web Consortium">W3C</abbr> Membership. This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.</p>
```

Comment:
```suggestion
              <p>Publication as a <em>Version 1.0.0-cr.1</em> does not imply endorsement by the <abbr title="World Wide Web Consortium">W3C</abbr> Membership. This document is a Candidate Recommendation by the W3C Solid Community Group and we invite implementations. It will only change in response to important implementation feedback. It is inappropriate to cite this document as other than work in progress.</p>
```